feedsOnChain.yaml

```diff
@@ -150,6 +150,34 @@ UMB-USD:
           fsym: UMB
           tsyms: USD
 
+# histo
+
+DAFI-USD-TWAP-1day:
+  discrepancy: 1.0
+  precision: 2
+  inputs:
+    - fetcher:
+        name: CoinmarketcapHistoHour
+        params:
+          symbol: DAFI
+          convert: USD
+          count: 25
+      calculator:
+        name: TWAP
+
+DAFI-USD-VWAP-1day:
+  discrepancy: 1.0
+  precision: 2
+  inputs:
+    - fetcher:
+        name: CoinmarketcapHistoDay
```

Comment:
It cannot be 1 day. you request 2 bars were each corresponds to 1 day.

I think it should be DAFI-USD-VWAP-1week, where count: 7.

Please review the requirement

Reply:
I added it as 1week (count=8) and I will double-check with the business